testsuite/end2end/profiles.yaml

```diff
@@ -0,0 +1,856 @@
+# This file defines discovery-related characteristics of management
+# profiles. It is used by pywbem end2end testcases.
+#
+# The format is a list of profile definition items, where each item describes
+# one profile. The order of profile definition items determines the order in
+# which the profiles are tested.
+#
+# Each profile definition item is a dictionary with the following items:
+# * registered_org: Registered organization of the profile (as a string, e.g.
+#   'SNIA').
+# * registered_name: Registered name of the profile.
+# * autonomous: Boolean indicating whether the profile is autonomous.
+# * central_class: Class name of the central class of the profile.
+# * scoping_class: Class name of the scoping class of the profile. May be
+#   null or non-null for autonomous profiles.
+# * scoping_path: Iterable of class names of the scoping path (from central to
+#   scoping, excluding central and scoping classes). Must be null for
+#   autonomous profiles.
+# * doc: Informal reference to the standards document describing the profile.
+#
+# TODO #1554: Add support for representing profile versions.
+# TODO #1552: Update profile definitions from additional input.
+
+# -
+#   # TODO 2018-12 AM: Clarify where the 'SMI-S' profile is described.
+#   registered_org: SNIA
+#   registered_name: SMI-S
+#   autonomous: TBD
+#   central_class: TBD
+#   scoping_class: TBD
+#   scoping_path: TBD
+#   doc: TBD
+-
+  registered_org: SNIA
+  registered_name: Array
+  autonomous: true
+  central_class: CIM_ComputerSystem
+  scoping_class: CIM_ComputerSystem
+  scoping_path: null
+  doc: SMI-S Block
+-
+  # Note: This profile has been deprecated in SMI-S 1.7, and the DMTF
```

Comment:
Actually that statement is incorrect since SNIA is bypassing 1.7 because is was so full of errors.  The  conclusion for 1.8 will be that either the  WBEM Server or Object manager profiles will be acceptable. This  was required since NOBODY implemented the WBEM Server profile and probably never will

Reply:
Ok. Needs to be corrected.

Reply:
Karl: SMI-S 1.8 is out and its Revision 3 still states that the SNIA Server profile is deprecated.

Reply:
DONE. Changed the respective note to:
```
  # Note: The SNIA Server profile has been deprecated in SMI-S 1.7, listing
  #       the DMTF WBEM Server profile as a replacement. There is an intention
  #       in the SMI-S community to change this again, allowing the SNIA
  #       Server profile and the DMTF WBEM Server profile to be implemented.
  #       However, as of the current SMI-S 1.8 Revision 3, the SNIA Server
  #       profile is still listed as deprecated.
```